astropy/units/si.py

```diff
@@ -225,13 +225,7 @@
 ###########################################################################
 # FREQUENCY
 
-def_unit(
-    ["Hz", "Hertz", "hertz"],
```

Comment:
I'm wondering if a PR to set up this one would be to make a DeprecatedAlias (or similar) so that `Hertz` can be phased out without breaking code?